exporter/schema_parser.go

```diff
@@ -1,39 +1,28 @@
 package exporter
 
 import (
-	"errors"
 	"fmt"
-	"os"
-	"regexp"
-	"strings"
-
+	prom "github.com/prometheus/client_golang/prometheus"
 	"gopkg.in/yaml.v2"
+	"os"
 )
 
 type ResSchema struct {
-	ResourceName     string            `yaml:"resource_name"`            // Static resource name
-	ResourcePath     string            `yaml:"resource_path"`            // Resource path
-	ResourceFilter   map[string]string `yaml:"filter,omitempty"`         // Filter to find resource instances with the required parameters
-	NameField        string            `yaml:"name_field"`               // Resource instance name
-	StatFields       []string          `yaml:"stat_fields,omitempty"`    // Statistics fields
-	StatFieldsRegExp string            `yaml:"stat_fields_re,omitempty"` // Regular expression for searching statistics fields
-	SkipFields       []string          `yaml:"skip_fields,omitempty"`    // Forced skip fields
-
-	resourceFilter *regexp.Regexp
-	statFields     map[string]struct{}
-	skipFields     map[string]struct{}
-}
-
-func (s *ResSchema) GetResourceFilter() *regexp.Regexp {
-	return s.resourceFilter
-}
-
-func (s *ResSchema) GetStatFields() map[string]struct{} {
-	return s.statFields
-}
-
-func (s *ResSchema) GetSkipFields() map[string]struct{} {
-	return s.skipFields
+	// PromMetricName Name of the metric to be created
+	PromMetricName string `yaml:"prom_metric_name"`
```

Comment:
I think here probably we need to have multiple metrics for a given resource. 

something like:
```
type ResourceMetric struct {
   name, labels, filter...
}

type Resource struct {
   Metrics []ResourceMetrics
}
```